### DISCUSSION TITLE
Estorno de Recibos Já Submetidos à AGT

### DISCUSSION THREAD

**Discussion by viegasfh** [assoft-portugal/SAF-T-AO#205] [Q&A] [answered]
Surgiu-me uma dúvida relativamente ao estorno de Recibos já submetidos à AGT. Se o recibo ainda não foi submetido, não há problema. Basta anulá-lo e ao reportar, na estrutura do recibo (Payment), teremos o PaymentStatus = 'A'. A título de exemplo,

```xml
<DocumentStatus>
    <PaymentStatus>A</PaymentStatus>
    <PaymentStatusDate>2022-06-28T10:34:26</PaymentStatusDate>
    <Reason>Erro no tipo de documento</Reason>
    <SourceID>Admin</SourceID>
    <SourcePayment>P</SourcePayment>
</DocumentStatus>
```
Contudo, se o recibo já foi submetido à AGT, como fazemos para estornar esse recibo, visto que já não pode ser anulado?

**Comment by cryptolopes** [comment, marked as answer]
Possivelmente, voltar a comunicar o documento com a alteração do `PaymentStatus`.

**Reply by viegasfh** [reply]
OK, obrigado! 

Presumo que o mesmo se aplique às pro-formas, certo?

**Comment by batalhadematos** [comment]
Boas, a solução (não testada) pode estar nos recibos do tipo RG (outros recibos). Para além do RC para os recibos normais, pode ser que seja possível reportar este tipo de recibos usando o tipo "RG". Não posso precisar mas creio que existir um post aqui ou no grupo do telegram sobre este tema.
